src/tasks/AndroidAppBuilder/Templates/monodroid.c

```diff
@@ -216,6 +223,22 @@ mono_droid_runtime_init (const char* executable, int managed_argc, char* managed
     appctx_values[0] = ANDROID_RUNTIME_IDENTIFIER;
     appctx_values[1] = bundle_path;
 
+    char* file_name = "output.pefile";
```

Comment:
Maybe use some define instead of a string literal so that AndroidAppBuilder and the RuntimeConfigParserTask can customize the name if we want?

Reply:
Fixed. Thanks!